src/pages/trails/ballots/list/BallotsList.vue

```diff
@@ -76,20 +73,14 @@ export default {
     ]),
     ...mapMutations("trails", ["resetBallots", "stopAddBallots"]),
 
-    async onLoad(reseted) {
+    async onLoad(status) {
       let scrollY = window.scrollY;
-      this.loading = true
       if (
-        (scrollY > this.startY && this.limit <= this.maxLimit) ||
-        reseted === true
+        (scrollY > this.startY && this.limit !== this.maxLimit) ||
+        status === true
       ) {
         this.$refs.infiniteScroll.resume();
-        // Start allways with a limit of 200 and then go +100 on next query
```

Comment:
this seems to be reverting [this pr](https://github.com/telosnetwork/app-telos-native/pull/72/files). Is this correct?